docs/reference/performance-benchmarks.md

```diff
@@ -0,0 +1,599 @@
+# Performance Benchmarks and Optimization Results
+
+## Overview
+
+This document tracks performance benchmarks, optimization implementations, and measured improvements for real-time SDR signal processing in rad.io.
+
+**Last Updated**: 2025-10-28
+
+## Executive Summary
+
+### Current Performance Status
+
+| Component | Current Performance | Target | Status |
+|-----------|-------------------|--------|--------|
+| FFT 2048 | ~3-5ms | <2ms | ✅ Meets minimum, target improvement |
+| FFT 4096 | ~8-10ms | <5ms | ✅ Meets minimum, target improvement |
+| Waterfall Rendering | 30-45 FPS | 60 FPS | ⚠️ Needs optimization |
```

Comment:
Is this still accurate? This is another reason these types of files with derived data don't belong in our repository!